packages/clay-data-provider/src/__tests__/LRUCache.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LRUCache} from '../LRUCache';
+
+let cache: LRUCache<string, {}>;
+
+describe('LRUCache', () => {
+	beforeEach(() => {
+		cache = new LRUCache(3);
+	});
+
+	afterEach(() => {
+		cache.reset();
+	});
```

Comment:
Delete this; you don't need to reset the object because you get a brand new object anyway each time.

Reply:
Oh yeah, I guess I just put it in my mind that `new LRUCache` would only run once... even if it's `beforeEach` 😂